examples/Atlas/+atlasControllers/@InstantaneousQPController/setupAndSolveQP.m

```diff
@@ -157,8 +157,8 @@
 %----------------------------------------------------------------------
 % Set up problem constraints ------------------------------------------
 
-qdd_lb = -inf(1, nq);
-qdd_ub = inf(1, nq);
+qdd_lb = -500 * ones(1, nq);
+qdd_ub = 500 * ones(1, nq);
```

Comment:
This restores the existing bounds in `AtlasQPController`. It may not actually be necessary.